src/test/scala/swiss/dasch/api/ProjectsEndpointSpec.scala

```diff
@@ -279,10 +282,32 @@ object ProjectsEndpointSpec extends ZIOSpecDefault {
       },
     )
 
+  private val projectsSuite = suite("/admin/projects/{shortcode}")(
+    test("DELETE erase should delete the projcet folder") {
+      val shortcode = ProjectShortcode.unsafeFrom("1111")
+      // given a project folder with a file exists
+      StorageService
+        .getProjectFolder(shortcode)
+        .map(_ / "as" / "df")
+        .tap(path => Files.createDirectories(path) *> Files.createFile(path / "asdf-test.txt"))
+        *> {
+          // when deleting the project via the api
+          val req = Request
+            .delete(URL(Root / "projects" / s"${shortcode.value}" / "erase"))
+            .addHeader("Authorization", "Bearer fakeToken")
+          for {
+            res              <- executeRequest(req)
+            folderWasRemoved <- StorageService.getProjectFolder(shortcode).flatMap(Files.exists(_)).negate
+          } yield assertTrue(res.status == Status.Ok, folderWasRemoved)
+        }
+    },
+  )
```

Comment:
nitpick: I suppose you could also add a test to check that it doesn't work when allowEarseProjects is false... but maybe not needed for such basic logic

Reply:
Unfortunately I do not have an idea how to write a simple spec which would only exchange the `Features` layer without introducing lots of duplication in the test code.

TLDR; Can do … though I am not sure if that test would be worth it.